### ISSUE TITLE
rrd4j graphs not working since merging #2661

### ISSUE THREAD

**Issue by hakan42** [openhab/openhab#3168]
I am using the OH2 builds from cloudbees.

I cannot see any graps in my rrd4j anymore. The last build that worked o.k., was # 324 which was compiled (as far as I can remember) before merging #2661. The error I am getting in my logs is:

```
2015-09-11 18:56:06 [WARN ] [.s.m.p.e.PersistenceExtensions:408  ] - There is no queryable persistence service registered with the name 'rrd4j'
```

Reverting the installation to the build # 324 allows rrd4j graphs again.

**Comment by kaikreuzer** [comment]
Are you sure that you have the rrd4j bundle within your new installation?
Can you do an "ss rrd4j" on the console and check if it is started?

**Comment by bigbasec** [comment]
I had the same problem.  Clean checkout and nothing is ever written to the etc/rrdj4 folder, so obviously charts don't work.  Used a binding build from a month or so back and it's working fine.

**Comment by hakan42** [comment]
Yes, the bundle is there and started. Interestingly, now my jpa persistence service is being queried for the historic data, whereas (with the same set of config files), the rrd4j service was being queried.

In my services.cfg, I have the following (never changed since my very first setup ages ago)

```
org.eclipse.smarthome.persistence:default=rrd4j
org.eclipse.smarthome.chart:provider=default
```

```
HypriotOS: openhab@pitaya in ~/2.0.0-331
$ ls -l addons/*rrd4j*
lrwxrwxrwx 1 openhab openhab 103 Sep 11 18:52 addons/org.openhab.persistence.rrd4j-1.8.0-20150908.070009-73.jar -> /home/openhab/2.0.0-331/__addons_dist/addons/org.openhab.persistence.rrd4j-1.8.0-20150908.070009-73.jar
```

```
osgi> ss rrd4j
"Framework is launched."


id	State       Bundle
173	ACTIVE      org.openhab.persistence.rrd4j_1.8.0.201509080702
```

**Comment by hakan42** [comment]
Also, @grzech1983 seems to have a similiar problem: https://github.com/openhab/openhab/issues/2661#issuecomment-139458512

**Comment by grzech1983** [comment]
Yep. I'm having similar issue:

https://github.com/openhab/openhab2/issues/378

with latest MySQL persistance addon used in OpenHAB 2.

**Comment by kaikreuzer** [comment]
This fix for #2661 was faulty. I have fixed it for rrd4j with #3169 (please confirm!).
I will reopen #2661 so that this can be fixed for other services as well.
